<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GDSlideShowDialog</class>
 <widget class="QDialog" name="GDSlideShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QWidget" name="preWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="GDPictureShowButton" name="pBtnPrevious">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="nextWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="GDPictureShowButton" name="pBtnNext">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="GDPictureStateButton" name="pBtnShow">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GDPictureShowButton" name="pBtnClose">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="GDAnimationShow" name="sildeShowWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <widget class="GDPreListWidget" name="preListWidget">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>6</y>
           <width>16777215</width>
           <height>110</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::ViewMode::IconMode</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GDPictureShowButton</class>
   <extends>QPushButton</extends>
   <header>gdpictureshowbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GDPictureStateButton</class>
   <extends>QPushButton</extends>
   <header>gdpicturestatebutton.h</header>
  </customwidget>
  <customwidget>
   <class>GDAnimationShow</class>
   <extends>QWidget</extends>
   <header>gdanimationshow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GDPreListWidget</class>
   <extends>QListWidget</extends>
   <header>gdprelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
